pkg/shell/nav.scss

```diff
@@ -797,14 +837,44 @@ $desktop: $phone + 1px;
 /* Navigation animation */
 
 @keyframes navHostsSlide {
-  0% { opacity: 0.25; transform: scaleY(0); }
-  25% { opacity: 0.5; transform: scaleY(0.25); }
-  100% { opacity: 1; transform: scaleY(1); }
+  0% {
+    opacity: 0.25;
+    transform: scaleY(0);
+  }
```

Comment:
This is a matter of opinion. IMHO it's over-splitting, the lines are short enough that a compact paragraph is easier to understand, and more importantly, visually compare. If you have a strong opinion about this, please move that into a separate commit, otherwise revert.